components/wmtk_components/CMakeLists.txt

```diff
@@ -2,4 +2,5 @@ add_subdirectory(delaunay)
 add_subdirectory(input)
 add_subdirectory(isotropic_remeshing)
 add_subdirectory(mesh_info)
-add_subdirectory(output)
\ No newline at end of file
+add_subdirectory(output)
+add_subdirectory(manifold_extraction)
```

Comment:
Just remove the empty line between the commands. It's a minor thing but it looks cleaner.